tests/e2e/e2e.go

```diff
@@ -183,20 +184,25 @@ func pollNonRetCmdStdOutForString(cmdStr string, timeout time.Duration, check fu
 	}
 }
 
+func odoCreateProject(projectName string) {
```

Comment:
Please remove this function to ```util.go```. Also feel free to move those function to ```util.go``` which are not using directly ginkgo package

Reply:
Moved